doc/source/whatsnew/v1.3.0.rst

```diff
@@ -1006,123 +999,123 @@ Period
-- Bug in :meth:`DataFrame.pivot_table` returning a ``MultiIndex`` for a single value when operating on and empty ``DataFrame`` (:issue:`13483`)
-- Allow :class:`Index` to be passed to the :func:`numpy.all` function (:issue:`40180`)
-- Bug in :meth:`DataFrame.stack` not preserving ``CategoricalDtype`` in a ``MultiIndex`` (:issue:`36991`)
-- Bug in :func:`to_datetime` raising error when input sequence contains unhashable items (:issue:`39756`)
-- Bug in :meth:`Series.explode` preserving index when ``ignore_index`` was ``True`` and values were scalars (:issue:`40487`)
-- Bug in :func:`to_datetime` raising ``ValueError`` when :class:`Series` contains ``None`` and ``NaT`` and has more than 50 elements (:issue:`39882`)
+- Bug in :meth:`DataFrame.append` returning incorrect dtypes when used with combinations of ``datetime64`` and ``timedelta64`` dtypes (:issue:`39574`)
+- Bug in :meth:`DataFrame.pivot_table` returning a :class:`MultiIndex` for a single value when operating on an empty DataFrame (:issue:`13483`)
+- :class:`Index` can now be passed to the :func:`numpy.all` function (:issue:`40180`)
+- Bug in :meth:`DataFrame.stack` not preserving ``CategoricalDtype`` in a :class:`MultiIndex` (:issue:`36991`)
+- Bug in :func:`to_datetime` raising an error when the input sequence contained unhashable items (:issue:`39756`)
+- Bug in :meth:`Series.explode` preserving the index when ``ignore_index`` was ``True`` and values were scalars (:issue:`40487`)
+- Bug in :func:`to_datetime` raising a ``ValueError`` when :class:`Series` contains ``None`` and ``NaT`` and has more than 50 elements (:issue:`39882`)
 
 Sparse
 ^^^^^^
 
-- Bug in :meth:`DataFrame.sparse.to_coo` raising ``KeyError`` with columns that are a numeric :class:`Index` without a 0 (:issue:`18414`)
+- Bug in :meth:`DataFrame.sparse.to_coo` raising a ``KeyError`` with columns that are a numeric :class:`Index` without a ``0`` (:issue:`18414`)
 - Bug in :meth:`SparseArray.astype` with ``copy=False`` producing incorrect results when going from integer dtype to floating dtype (:issue:`34456`)
-- Implemented :meth:`SparseArray.max` and :meth:`SparseArray.min` (:issue:`40921`)
+- Bug in :meth:`SparseArray.max` and :meth:`SparseArray.min` would always return an empty result (:issue:`40921`)
 
 ExtensionArray
 ^^^^^^^^^^^^^^
 
-- Bug in :meth:`DataFrame.where` when ``other`` is a :class:`Series` with :class:`ExtensionArray` dtype (:issue:`38729`)
-- Fixed bug where :meth:`Series.idxmax`, :meth:`Series.idxmin` and ``argmax/min`` fail when the underlying data is :class:`ExtensionArray` (:issue:`32749`, :issue:`33719`, :issue:`36566`)
-- Fixed a bug where some properties of subclasses of :class:`PandasExtensionDtype` where improperly cached (:issue:`40329`)
-- Bug in :meth:`DataFrame.mask` where masking a :class:`Dataframe` with an :class:`ExtensionArray` dtype raises ``ValueError`` (:issue:`40941`)
+- Bug in :meth:`DataFrame.where` when ``other`` is a Series with :class:`ExtensionArray` dtype (:issue:`38729`)
```

Comment:
why removed `:class:` from Series? no real objection, just not clear on if i should change my habits when writing these

Reply:
This originated from https://github.com/pandas-dev/pandas/pull/38002#discussion_r528531153 and I agree with @jorisvandenbossche's assessment - links on Series and DataFrame serve only as a distraction in cases such as these. I think a good general rule is to only link when Series/DataFrame are the main subject of the change.